yarn-project/aztec-rpc/src/aztec_rpc_http/aztec_rpc_http_server.ts

```diff
@@ -23,9 +24,12 @@ export const localAnvil = foundry;
 
 /**
  * Wraps an instance of the Aztec RPC Server implementation to a JSON RPC HTTP interface.
+ *
+ * TODO rename this as part of #2451
+ *
  * @returns A new instance of the HTTP server.
  */
-export function getHttpRpcServer(aztecRpcServer: AztecRPC): JsonRpcServer {
+export function createAztecRpcRpcServer(aztecRpcServer: AztecRPC): JsonRpcServer {
```

Comment:
umm did we maybe mean 'httpRpc' rather than `rpcRpc`? @benesjan

Reply:
I've undone the change in this PR. Renaming these functions should be done as part of #2451